### PR TITLE
Add file with package versions for external packages

### PR DESCRIPTION
**Pull request by chcosta**
Companion change for https://github.com/dotnet/buildtools/pull/748

/cc @weshaggard @dagood @jhendrixMSFT

### CODE REVIEW

File: pkg/ExternalPackages/versions.props [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
+  <ItemGroup>
+    <ExternalPackage Include="System.IO.Compression">
+      <Version>4.1.0-rc3-24117-00</Version>
```
Comment (weshaggard) [inline pkg/ExternalPackages/versions.props]: We should make sure we have work tracking how we plan to update these versions.

Comment (chcosta) [inline pkg/ExternalPackages/versions.props]: I mentioned that in my hand-off to @jhendrixMSFT , but I'll file an issue to make sure it doesn't get lost.

**Comment by chcosta** [comment]
@jhendrixMSFT , I've merged https://github.com/dotnet/buildtools/pull/748, after that gets published, you'll need to update BuildToolsVersions.txt in CoreFx before you can merge this change.

**Comment by jhendrixMSFT** [comment]
The build tools changes are in.

**Comment by dagood** [comment]
I'm late on this PR, but could we tie this versions props file into dependency validation for the split between TFS vs. CoreFX? (Or is that tie-in the plan?) It looks like validation could be modified pretty simply to take these as inputs for specific packages.

**Pull request merged**